test/e2e/parallel/spacerequest_test.go

```diff
@@ -110,14 +145,14 @@ func TestCreateSpaceRequest(t *testing.T) {
 	t.Run("subSpace has parentSpace target cluster when target roles are empty", func(t *testing.T) {
 		// when
 		spaceRequest, parentSpace := CreateSpaceRequest(t, awaitilities, memberAwait.ClusterName,
-			WithSpecTierName("appstudio"))
+			WithSpecTierName("appstudio-env"))
 
 		// then
 		// check for the subSpace creation with same target cluster as parent one
 		subSpace, err := awaitilities.Host().WaitForSubSpace(t, spaceRequest.Name, spaceRequest.Namespace, parentSpace.GetName(),
 			UntilSpaceHasTargetClusterRoles([]string(nil)),                     // empty target cluster roles
 			UntilSpaceHasStatusTargetCluster(parentSpace.Status.TargetCluster), // subSpace should have same target cluster as parent space
-			UntilSpaceHasTier("appstudio"),
+			UntilSpaceHasTier("appstudio-env"),
 			UntilSpaceHasAnyProvisionedNamespaces(),
 		)
 		require.NoError(t, err)
```

Comment:
when verifying this, you can also check the `UntilSpaceRequestHasNamespaceAccess()`

Reply:
It should be done in https://github.com/codeready-toolchain/toolchain-e2e/pull/706/commits/3f315e55a25920ece38c2351288e4faa2176ae66